core/tests/performance_framework/perf_services.py

```diff
@@ -17,11 +17,13 @@
 """
 
 import os
+import random
 import time
 import urlparse
 from browsermobproxy import Server
```

Comment:
These two should go below the internal imports. See main.py for an example.

So:

```
import os
import random
import time
import urlparse

from core.tests.performance_framework import perf_domain
import feconf

from browsermobproxy import Server
from selenium import webdriver
```